xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -377,10 +479,13 @@
         <Parameter Name="unmanagedDllPath" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="unmanagedDllPath">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="unmanagedDllPath">The path to the unmanaged DLL</param>
+        <summary>Loads an unmanaged DLL from the specified path.</summary>
+        <returns>Pointer to the loaded DLL.</returns>
```

Comment:
The return value should not be treated as a pointer. It's a "handle". On Windows it's the `HMODULE`, not sure what it stands for on Linux and Mac. I know that `HMODULE` on Windows really is a pointer, but I view that only as an implementation detail. Definitely the docs for Win32 APIs around this never refer to the returned value as a pointer/address.

Reply:
"A pointer", or "A handle"

Reply:
Linux `dlopen` docs refer to the returned `void *` as a handle.  So `A handle` seems best.